games/devtest/mods/unittests/password_hash.lua

```diff
@@ -0,0 +1,25 @@
+local function test_password_hash()
+	local password = "hunter2"
+	local user = "Cthon98"
+	
+	local hash = core.get_password_hash(user, password)
+	local hash_no_pass = core.get_password_hash(user, "")
+	assert(hash)
+	assert(hash_no_pass)
```

Comment:
not sure what the intent here is
```suggestion
	assert(type(hash_no_pass) == "string" and hash_no_pass ~= "")
```